reference/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS.md

```diff
@@ -42,16 +42,16 @@ brew cask upgrade powershell
 > [!NOTE]
 > The commands above can be called from within a PowerShell (pwsh) host,
 > but then the PowerShell shell must be exited and restarted to complete the upgrade
-> and refresh the values shown in $PSVersionTable.
+> and refresh the values shown in `$PSVersionTable`.
 
 [brew]: http://brew.sh/
 
 ## Installation of latest preview release via Homebrew on macOS 10.12 or higher
 
 See [About Brew](#about-brew) for information about Brew.
 
-Once you've installed Homebrew, installing PowerShell is easy.
-First, install [Cask-Versions][cask-versions] which lets you install alternative versions of cask packages:
+After you've installed Homebrew, you can install PowerShell.
+First, install the [Cask-Version][cask-versions] that lets you install alternative versions of cask packages:
```

Comment:
The name is `Casks-Versions`.  It seems like we should stick with their name.

Reply:
@TravisEz13 Yes, thanks we are fixing that.